src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -103,23 +111,9 @@ def generate_MCP_1_default_config(self, filename):
     }
 
     conditionals = [conditionals_device_2, conditionals_device_4, conditionals_device_8]
-    # Remove conditionals from registration
```

Comment:
Why are you undoing this?

Reply:
we need those fields in reference models calculations had to remove the deletions.